codalab_service.py

```diff
@@ -213,6 +219,30 @@ def has_callable_default(self):
     CodalabArg(name='use_ssl', help='Use HTTPS instead of HTTP', type=bool, default=False),
     CodalabArg(name='ssl_cert_file', help='Path to the cert file for SSL'),
     CodalabArg(name='ssl_key_file', help='Path to the key file for SSL'),
+    ### Worker manager
+    CodalabArg(name='worker_manager_type', help='Type of worker manager (e.g., aws, azure, slurm)'),
+    CodalabArg(
+        name='worker_manager_cpu_queue',
+        help='Name of queue to submit CPU jobs',
+        default='codalab-cpu',
+    ),
+    CodalabArg(
+        name='worker_manager_gpu_queue',
+        help='Name of queue to submit GPU jobs',
+        default='codalab-gpu',
+    ),
+    CodalabArg(
+        name='worker_manager_max_cpu_workers',
+        help='Maximum number of CPU workers',
```

Comment:
This is the maximum number of CPU workers _per_ worker manager (not overall), right?

Reply:
Clarified.